src/Storages/ProjectionsDescription.cpp

```diff
@@ -286,7 +286,7 @@ Block ProjectionDescription::calculate(const Block & block, ContextPtr context)
                                                                        : QueryProcessingStage::WithMergeableState})
                        .buildQueryPipeline();
     builder.resize(1);
-    builder.addTransform(std::make_shared<SquashingChunksTransform>(builder.getHeader(), block.rows(), block.bytes()));
+    builder.addTransform(std::make_shared<SquashingChunksTransform>(builder.getHeader(), block.rows(), 0));
```

Comment:
so, what was the problem?)

Reply:
The original code is actually wrong (aggregated block might contain more bytes). However, without external grouping, there will always be one block, which never triggers the bug)

Reply:
@amosbird can you put a comment? This does not looks obvious at first glance.

Reply:
```suggestion
    // Generate aggregated blocks with rows less or equal than the original block.
    // There should be only one output block after this transformation.
    builder.addTransform(std::make_shared<SquashingChunksTransform>(builder.getHeader(), block.rows(), 0));
```